connect-id-example/src/main/java/com/telenor/connect/connectidexample/SignInActivity.java

```diff
@@ -3,20 +3,23 @@
 import android.app.Activity;
 import android.content.Intent;
 import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
 import android.util.Log;
 
 import com.telenor.connect.ConnectCallback;
 import com.telenor.connect.ConnectSdk;
 import com.telenor.connect.ui.ConnectLoginButton;
 import com.telenor.connect.utils.ConnectUtils;
 
-public class SignInActivity extends Activity {
+public class SignInActivity extends AppCompatActivity {
+
+    ConnectLoginButton loginButton;
```

Comment:
Do we need this change?

Reply:
I.e. the hoisting of `loginButton`.

Reply:
I guess it can move it from here, it's used just in 1 place